homeworks/gaalistvantamas/5_oop/car_fleet_mgr.py

```diff
@@ -0,0 +1,106 @@
+"""
+Author: Gaál István Tamás
+Task: Homework-5
+"""
+
+class Car:
+
+    FUEL_USAGE = 0.1
+    
+    def __init__(self, brand, model, year, mileage = 0, fuel_level = 100.0):
+        self.brand = brand
+        self.model = model
+        self.year = year
+        self.mileage = mileage
+        self.fuel_level = fuel_level
+
+    def drive(self, road_length_in_kilometres):
+        if road_length_in_kilometres < 1:
+            print("Please give a bigger number than 0!")
+        
+        elif not road_length_in_kilometres > int(self.fuel_level / self.FUEL_USAGE):
+            self.mileage += road_length_in_kilometres
+            self.fuel_level -= road_length_in_kilometres * self.FUEL_USAGE
+            print(f"You drove {road_length_in_kilometres} km and the fuel usage of the trip was {round(100 - self.fuel_level)}%!")
+
+        else:
+            print("There is no enough fuel in the car!")
+            print(f"You can drive maximum: {self.fuel_level / self.FUEL_USAGE} km!")
+
+    def refuel(self, refuel_percents):
+        if refuel_percents < 1:
```

Comment:
Itt a hibaüzenet és a feltétel nincs összhangban. Ha megadom hogy tankoljon 0.5%-nyit, hibát kapok hogy 0-nál nagyobb számot adhatok csak meg. Vagy az legyen hogy refuel_percents `<= 0`
vagy a hibaüzenet legyen az hogy `"The minimum amount for refueling is 1 %"` vagy valami hasonló.